tests/unit/type/list.tcl

```diff
@@ -412,6 +463,29 @@ start_server {
         assert_equal {foo} [r lrange target2{t} 0 -1]
     }
 
+    test "BLMPOP with multiple blocked clients" {
+        set rd1 [redis_deferring_client]
+        set rd2 [redis_deferring_client]
+        set rd3 [redis_deferring_client]
+        set rd4 [redis_deferring_client]
+        r del blist{t} blist2{t}
+
+        $rd1 blmpop 2 blist{t} blist2{t} left count 1 0
+        $rd2 blmpop 2 blist{t} blist2{t} right count 10 0
+        $rd3 blmpop 2 blist{t} blist2{t} left count 10 0
+        $rd4 blmpop 2 blist{t} blist2{t} right count 1 0
+
+        r lpush blist{t} a b c d e
+        r lpush blist2{t} 1 2 3 4 5
```

Comment:
let's use multi-exec to be sure both lists became available at the same event loop